pkg/provider/provider.go

```diff
@@ -230,6 +308,12 @@ func (k *kubeProvider) DiffConfig(ctx context.Context, req *pulumirpc.DiffReques
 	if olds["enableDryRun"] != news["enableDryRun"] {
 		diffs = append(diffs, "enableDryRun")
 	}
+	if olds["renderYamlToDirectory"] != news["renderYamlToDirectory"] {
+		diffs = append(diffs, "renderYamlToDirectory")
+
+		// If the render directory changes, all of the manifests will be replaced.
+		replaces = diffs
```

Comment:
Is this the right way to do this?  This will end up placing all props that were diffs into replaces.  Don't we want to just add `renderYamlToDirectory`?

Reply:
Updated